default.vcl

```diff
@@ -68,14 +68,23 @@ sub vcl_recv {
     # Remove all cookies; we don't need them, and setting cookies bypasses varnish caching.
     unset req.http.Cookie;
 
-    # allow PURGE from localhost
+    # allow PURGE from localhost and 10.2...
     if (req.method == "PURGE") {
```

Comment:
Do we still need purge if we have ban?

Reply:
The purge block was in there before my changes. The only change I've made regarding the purging code was to enhance the `acl purge` in order to allow access from the pods inside the cluster. But this `acl purge` is also used for the banning block.

Reply:
:+1: